src/regfunc.jl

```
@@ -0,0 +1,349 @@
+#-----------------------------------------------------------
+#-----------------------------------------------------------
+# Define function to access windows registry
+# -- part of WinReg Module --
+#
+#File: regutils.jl
+#Author: Leonardo Cocco
+#Creation Date: 11-04-2016
+#Last update 21-11-2016
+#ver: 0.11
+#----------------------------------------------------------
+#----------------------------------------------------------
+
+
+"""Read a value with `valuename` into `key`. 
+`key` can be specified using abbreviated form (e.g. 'HKEY_LOCAL_MACHINE' -> 'HKLM').
+Return type in Julia is based onto value type stored in registry; returns `nothing` (type is `Void`) if key not found"""
+function regread(key::AbstractString, valuename::AbstractString)
+	local root = getroothkey(key)
+	if root == 0
+		return nothing
```

I think we should throw errors rather than return `nothing`.

---

OK